### ISSUE TITLE
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

### ISSUE THREAD

**Issue by belykh** [storybookjs/storybook#14160]
**Describe the bug**
When I added addon `@storybook/addon-a11y` I've got a warning after running and building storybook

package.json
```
{
  "name": "test",
  "version": "1.0.0",
  "scripts": {
    "storybook": "start-storybook -p 6006 --no-dll",
    "build-storybook": "build-storybook --no-dll"
  },
  "devDependencies": {
    "@storybook/addon-a11y": "^6.1.21",
    "@storybook/addon-actions": "^6.1.21",
    "@storybook/addon-essentials": "^6.1.21",
    "@storybook/vue": "^6.1.21",
    "vue": "^2.6.12",
    "vue-loader": "^15.9.6"
  }
}
```

commands:
```
npm i
npm run storybook
```

building log (partially):
```
...
WARNING in ./node_modules/axe-core/axe.js 6275:19-26
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./node_modules/@storybook/addon-a11y/dist/a11yRunner.js
 @ ./node_modules/@storybook/addon-a11y/dist/a11yRunner.js-generated-other-entry.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-docs/dist/frameworks/common/config.js-generated-other-entry.js ./node_modules/@storybook/addon-docs/dist/frameworks/vue/config.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/preset/addArgs.js-generated-other-entry.js ./node_modules/@storybook/addon-backgrounds/dist/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-backgrounds/dist/preset/addParameter.js-generated-other-entry.js ./node_modules/@storybook/addon-a11y/dist/a11yRunner.js-generated-other-entry.js ./node_modules/@storybook/addon-a11y/dist/a11yHighlight.js-generated-other-entry.js ./.storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined
...
```

**System**
```
Environment Info:

  System:
    OS: Windows 10 10.0.19042
    CPU: (8) x64 Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
  Binaries:
    Node: 14.15.4 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.14.5 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 89.0.4389.72
    Edge: Spartan (44.19041.423.0), Chromium (89.0.774.45)
  npmPackages:
    @storybook/addon-a11y: ^6.1.21 => 6.1.21
    @storybook/addon-actions: ^6.1.21 => 6.1.21
    @storybook/addon-essentials: ^6.1.21 => 6.1.21
    @storybook/vue: ^6.1.21 => 6.1.21
```

**Comment by shilman** [comment]
Did you try upgrading to 6.2-beta?

```
npx sb@next upgrade --prerelease
```

**Comment by belykh** [comment]
Checked on 6.2-beta.11

No warnings on this version
Confirmed

Thank you!

**Comment by victoriatomzik** [comment]
These subpackages are still not upgraded: (6.2.0-rc.9)
<img width="866" alt="image" src="https://user-images.githubusercontent.com/17315148/112465915-26f50980-8d5d-11eb-827f-14236550ea86.png">

**Comment by shilman** [comment]
@victoriatomzik can you explain how this is related to the issue?
